src/main/java/TestsService.java

```diff
@@ -22,8 +24,9 @@ public TestsService(TaskDao taskDao) {
     public PerformedTask submitTask(long studentId, long taskId, double calculatedResult) {
         logger.info("Submitting task for studentId {} taskId {} and calculated result {}", studentId, taskId, calculatedResult);
         Task task = taskDao.getById(taskId);
-        boolean isPassed = calculateIfResultIsCorrect(task.getExpectedResult(), calculatedResult, task.getMarginOfError());
+        boolean isPassed = calculateIfResultIsCorrect(task.getExpectedResult(), calculatedResult, task.getMarginOfError()); // Pytanie! Dlaczego nie przekazujemy poprostu całego obiektu klasy task, tylko wartości double jako wyniki metod klasy task? Wtedy przekazywalibyśmy dwa argumenty zamiast trzech
```

Comment:
można tak zrobić, chyba nawet będzie lepiej (to jest właśnie reformatowanie kodu)